schemata/contentblock.yaml

```diff
@@ -55,9 +55,21 @@ properties:
   metadata:
     "$ref": "metadata.json"
   created_at: # FIXME: ISOdate,null
-    type: number
+    type: string
+    format: 'date-time'
   updated_at: # FIXME: ISOdate,null
-    type: [number, "null"]
-  anyOf: [headline, text, cta, media, list]
+    type: [string, "null"]
+    format: 'date-time'
+anyOf: [
+  '$ref': 'headline.json'
+,
```

Comment:
It is arguable which syntax is prettier. But anyway, the main point here is to get back to point where we're actually testing things